Governance/retire/retire-replica.md

```diff
@@ -11,23 +11,23 @@ This is because it is hard in the first place to choose which data, or data subs
 
-Some methods are machine-driven, where you use any available statistics automatically saved by the data server or filesystem. These can be useful when managing a big dataset where you have set rules from the start, so that ongoing retirement of data subsets are anticipated and part of the regular management. Here we called these "automated methods".
+Some methods are machine-driven, where any available statistics automatically saved by the data server or filesystem are used. These can be useful when managing a big dataset where there are set rules from the start, so that ongoing retirement of data subsets are anticipated and part of the regular management. Here these are called "automated methods".
 
-Even if you have a reliable automatic measure of data access it is always a good idea to consult the community. Automated methods can only record actual usage, to make an informed decision you also need to assess future and other potential usage. It is also important to have an idea of how the data is used, if an obscure variable is needed to produce a dataset that is widely used by the community, deciding to retire it because there is only one person using it is not a good idea.
+Even if there is a reliable automatic measure of data access, it is always a good idea to consult the community. Automated methods can only record actual usage, to make an informed decision future needs and other potential usages should be assessed. It is also important to have an idea of how the data is used, if an obscure variable is needed to produce a dataset that is widely used by the community, deciding to retire it because there is only one person using it is not a good idea.
 
 ### Automated methods
-* **Checking latest access time** (`atime`) of the files (using the `stat` command) to see when a file was last accessed. Note that many operations can affect `atime` so it's important to do this before moving the data anywhere. This is only reliable if there are not automated processes that alter `atime` when running, for example a regular scan of the filesystem run by the system administrator. This can only tell you when a file was last accessed not how many users are accessing it. 
-* **Project user number:** where users need to sign up to use the data (as for NCI project) you can easily get an idea of users' number and distribution. A limit of this is that it doesn't tell you who is actively using the data or which subset of the data is being used.
+* **Checking latest access time** (`atime`) of the files (using the `stat` command) to see when a file was last accessed. Note that many operations can affect `atime` so it's important to do this before moving the data anywhere. This is only reliable if there are not automated processes that alter `atime` when running, for example a regular scan of the filesystem run by the system administrator. This can only say when a file was last accessed not how many users are accessing it. 
```

Comment:
```suggestion
* **Checking latest access time** (`atime`) of the files (using the `stat` command) to see when a file was last accessed. Note that many operations can affect `atime` so it's important to do this before moving the data anywhere. This is only reliable if there are not automated processes that alter `atime` when running, for example a regular scan of the filesystem run by the system administrator. This can only indicate when a file was last accessed not how many users are accessing it. 
```